src/utils.js

```diff
@@ -560,6 +561,35 @@ const htmlToText = (html) => {
     return text.trim();
 };
 
+/**
+ * Creates a standardized debug info from request and response. This info is usually added to dataset under the hidden `#debug` field.
+ *
+ * @param {Object} request [Apify.Request](https://sdk.apify.com/docs/api/request) object.
+ * @param {Object} [response] Puppeteer [Response](https://pptr.dev/#?product=Puppeteer&version=v1.11.0&show=api-class-response) object
+ * or NodeJS [http.ServerResponse](https://nodejs.org/api/http.html#http_class_http_serverresponse) object
+ * @param {Object} [additionalFields] Object containing additional fields to be added.
+
+ * @return {Object}
+ * @memberOf utils
+ */
+const createRequestDebugInfo = (request, response = {}, additionalFields = {}) => {
```

Comment:
IMHO we should flag this function @ignore, it's quite internal, it will only confuse users and if somebody starts using it, we won't be able to change it